paper_trail.gemspec

```diff
@@ -26,7 +26,10 @@ has been destroyed.
   s.require_paths = ["lib"]
 
   s.required_rubygems_version = ">= 1.3.6"
-  s.required_ruby_version = ">= 2.3.0"
+
+  # Ruby 2.4 reaches EoL at the end of March of 2020
```

Comment:
Would we bump this to 2.5 at the end of March?

Reply:
We could, or we could wait a while. Supporting 2.4 does make our lives a little harder because rails 6 doesn't (see eg. `travis.yml`). But, it's not a big deal.